packages/ui/src/Toast/open.ts

```diff
@@ -72,7 +76,10 @@ export const openToast = (props: ToastProps): Toast => {
 
     id = fn(options.message, {
       id,
-      description: options.description,
+      description:
+        typeof options.description === 'string'
+          ? truncateString(options.description, 200)
+          : options.description,
```

Comment:
issue: i don't think this should be a part of the UI component. If users need this, they can truncate the string in their code when passing `description` to the Toast. Moreover, UI components should not trim the string programmatically – this worsens the UX. Think of it as a long but important error message – what might happen if users see the start but not the important end of the message? 

From the UI components part, we could add "show more"/"show less" buttons if this is really necessary. But still, the description is defined as `ReactNode`, so this can be done on the users' side.

Reply:
I suppose you're right, but I have no idea where this long error message throwIfAborted comes from.